tests/utils/handler/cluster_handler.go

```diff
@@ -816,7 +816,7 @@ func (ch *clusterHandler) GenerateClusterCreateFlags() ([]string, error) {
 		)
 		if ch.profile.ClusterConfig.AllowedRegistries {
 			flags = append(flags,
-				"--registry-config-allowed-registries", "allowed.example.com,*.test",
+				"--registry-config-allowed-registries", "allowed.example.com,quay.io,*.redhat.com",
```

Comment:
Could you add a condition by 'else if' for ch.profile.ClusterConfig.BlockedRegistries, rather consider ch.profile.ClusterConfig.BlockedRegistries= true if ch.profile.ClusterConfig.AllowedRegistries = false, as BlockedRegistries and AllowedRegistries are two separated config in profile

Reply:
Thanks.Update it